CONTRIBUTING.md

```diff
@@ -1,196 +1,208 @@
-
-#### [discussion](https://github.com/OperationCode/operationcode/labels/discussion)
-discussions will generally not have very specific actionable items in them and can be used to plan and design a feature.
-
-#### [milestones](https://github.com/OperationCode/operationcode/milestones)
-Milestones are used as containers for new issues pertaining to a project, time frame or feature.
-
-#### [beginner-friendly](https://github.com/OperationCode/operationcode/labels/beginner%20friendly)
-These items are hand picked as being great candidates for your first issue to work on.
+1. [Setting Up Your Environment](#setting-up-your-environment)
+2. [Find an Issue To Work On](#finding-an-issue)
+3. [Submit Your Pull Request](#submitting-your-changes)
 
 ## Setting Up Your Environment
-In order to work on the Operation Code site you'll need to install a few things. Beginner's Note: If you dont have Ruby and Rails installed in your system, dont worry about it. They will get installed during the bundle command, after you clone the repo, along with all the other dependencies.  
-
-### Ruby
-Ruby is an Object Oriented programming language Operation Code is written in. While you can install ruby directly to your machine it's highly recommended to use a version manager.
-
-* [Install rvm](https://rvm.io/rvm/install)
-* Look [here](https://raw.githubusercontent.com/OperationCode/operationcode/master/.ruby-version) to find the version of ruby we're using
-* Install the version of ruby from the step above. eg `rvm install 2.2.3`
+* In order to work on the **Operation Code** site, you will need to install a few things. 
+  * Beginner's Note: If you don't have Ruby and Ruby on Rails installed in your system yet, don't worry about it. After you clone the GitHub repository, they will be installed during the `bundle` command along with all the dependencies.  
+
+  ### Ruby
+  Ruby is an Object-Oriented programming language that **Operation Code** is written in. While you can install Ruby directly to your machine, it is highly recommended to use a version manager. We recommend RVM, but rbenv will also work.
+
+  * [Install RVM](https://rvm.io/rvm/install)
+  * Look [here](https://raw.githubusercontent.com/OperationCode/operationcode/master/.ruby-version) to find the version of Ruby we are using
+  * Install the version of Ruby from the step above. Example: `rvm install 2.3.0`
+
+  ### Git
+  Git is a distributed version control system. This is how our code is stored and managed. Git can be frustrating, but it is an essential tool. If you want to learn more about Git, a great resource is [Think Like a Git](http://think-like-a-git.net/). If you find yourself in a real git pickle, see ["Oh, shit, git!"](http://ohshitgit.com/).
+
+  * [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+  ### PostgreSQL
+  PostgreSQL is an open source Object-Relations database. This stores all of **Operation Code**'s data.
```

Comment:
`PostgreSQL is an open source Object-Relations database`

I think you mean Object-Relational?